app/views/application/_sidebar.html.haml

```diff
@@ -6,6 +6,11 @@
 
       = link_to 'https://discord.gg/7E6kC2H', target: '_blank', rel: 'noopener', class: 'btn btn-success pull-right' do
         Join our Discord
+    %li.list-group-item
+      = image_tag 'patreon-logo.png'
+
+      = link_to 'http://steamcommunity.com/groups/ozfortress', target: '_blank', rel: 'noopener', class: 'pull-right' do
```

Comment:
Pretty sure we don't want to link to the steam group here.

Reply:
Whoops, thats a yikes...